rust/processor/src/db/common/models/fungible_asset_models/v2_fungible_asset_balances.rs

```diff
@@ -394,6 +394,7 @@ impl FungibleAssetBalance {
                     last_transaction_timestamp: txn_timestamp,
                     token_standard: TokenStandard::V1.to_string(),
                 };
+                // What is this for?
```

Comment:
V1 events don't have asset type. It's just "0x1::coin::WithdrawEvent" (or something like that) w/ the address and amount of APT.

Reply:
Ok I see you've commented about it in the main file so I'll leave this.